### ISSUE TITLE
[Feature]: Please list changes in app whenever it updates so we know what to look for

### ISSUE THREAD

**Issue by asciiletters** [Memmy-App/memmy#1026]
### Check for open issues

- [X] I have verified that another issue for this is not open, or it has been closed and has not been properly implemented.

### What is the feature that you would like to have added?

It looks like you are doing a lot of work to the app. Some of it feels like we went backwards from where we were. It would help to get a change log explaining each update. 

### Solution

Some devs provide that as a notification announcement page that appears after an update next time the app is is opened. 

### Alternatives

None. You instead have testers unsure of what or why things have changed.

**Comment by gkasdorf** [comment]
Yea I'll be doing that. It's just been a quick pace past few days and I have not completely gotten the CI/CD pipeline set up correctly yet to reflect those. Trying to automate the release process so I don't have to spend my time waiting on builds.

I'll see if I can't get the commit messages between releases to be the changelog 👍
